### DISCUSSION TITLE
Change version number if there is an update please.

### DISCUSSION THREAD

**Discussion by x2611** [rustdesk/rustdesk#6284] [General]
I constantly check for RD updates. The last three updates were all named  rustdesk-1.2.3-x86_64.
The only way I could tell it was new was to download it, check size difference and then run it to get the update button.

Please add a sub version number in the new file name so we can tell there is something new.
An auto update feature would be even better. Or am I missing something?

Thx.

**Comment by fragtion** [comment]
Auto update is would be really good (particularly for nightly). Or if not automatic, it should be possible to invoke the update manually from the client connection (or the host gui - without needing to come to github and download the installer packages manually)
The new features are often worth updating, then a manual update is needed for entire fleet..
